CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Welcome!
+Thank you for your interest in Dashing! This exceptionally handsome
+framework welcomes all input, but being stylish requires certain protocols be followed. :bowtie:
+
+Below you will find a set of guidelines that will ensure the best outcome for you _and_ Dashing.
+<a name="issues"></a>
+## Have an Issue
+If you run into problems with Dashing, please take these steps before submitting an issue:
+
+1. Check the [Troubleshooting Guide]() in the wiki.
+2. Use the [GitHub Issue Search](https://help.github.com/articles/searching-issues/) to check if the issue has already been reported.
+3. Submit your issue to the [best place](). Please provide as much helpful information as possible, preferably making use of a [reduced test case](https://www.google.ca/search?#q=reduced%20test%20case).
+
+**Support requests should be directed to [Stack Overflow](http://stackoverflow.com/questions/tagged/dashing).**
+<a name="features"></a>
+## Feature Requests
+Feature requests are welcome, but take a moment and consider whether your idea
+fits with the scope and aims of the project. A good rule of thumb is to apply the 80/20 rule: every feature should be useful to at least 80% of Dashing. Dashing has many use cases, and it's impossible to accommodate every edge case.
+
+If you feel that you have a really amazing, super neato idea that doesn't quite fit with the core use of Dashing, it may be a good candidate for an external Gem which supercharges a project. An excellent example of this is [dashing-contrib](https://github.com/QubitProducts/dashing-contrib).
+<a name="pull-requests"></a>
+## Pull Requests
+
+Patches, improvements and new features are a fantastic
+help -- thank you!
+
+**Please ask first** before embarking on any significant pull request (e.g.
+implementing features, refactoring code, porting to a different language),
+otherwise you risk spending a lot of time working on something that may not be merged into the project.
+
+Please adhere to the coding conventions used throughout a project (indentation,
+accurate comments, etc.) and any other requirements (such as test coverage). All code submitted via Pull Request will be dicussed and critiqued in a respectful manner. This is not an attack on you or your code, it is simply a means of ensuring that Dashing remains exceptionally handsome (inside and out).
+
+GitHub has [excellent documentation on how to use Pull Requests.](https://help.github.com/articles/using-pull-requests/)
+
+**IMPORTANT**: By submitting a patch, you agree to allow the project owner to
+license your work under the same license as that used by the project.
+<a name="commit-msgs"></a>
+## Git Commit Message Suggestions
+* Use the present tense ("Add feature" not "Added feature")
```

Comment:
Yeah, although it's nice to have consistency with verb tenses and moods, I don't want to make the list of "rules"/"suggestions" seem too long. I'd just stick with the emojis